packages/dds/tree/src/checkout/checkout.ts

```diff
@@ -17,5 +17,5 @@ export interface ICheckout extends IForestSubscription {
      * TODO: support nesting (perhaps via "commands"),
      * and do this in a way where there is control over which ones intermediate versions are displayed.
      */
-    runTransaction<T>(f: (transaction: IEditableForest) => T): T;
+    runTransaction<T>(transaction: (forest: IEditableForest) => T): T;
```

Comment:
Consider updating the doc comment at well. For some reason I can't make suggestions for that as part of the CR.